e2e/testcafe-devextreme/tests/scheduler/common/layout/horizontalScroll/horizontalScroll.ts

```diff
@@ -0,0 +1,169 @@
+    startDate: new Date('2021-03-31T16:45:00.000Z'),
+    endDate: new Date('2021-03-31T18:15:00.000Z'),
+  }, {
+    text: 'Approve New Online Marketing Strategy',
+    startDate: new Date('2021-03-31T19:00:00.000Z'),
+    endDate: new Date('2021-03-31T21:00:00.000Z'),
+  }, {
+    text: 'Upgrade Personal Computers',
+    startDate: new Date('2021-03-31T22:15:00.000Z'),
+    endDate: new Date('2021-03-31T23:30:00.000Z'),
+  }, {
+    text: 'Customer Workshop',
+    startDate: new Date('2021-04-01T18:00:00.000Z'),
+    endDate: new Date('2021-04-01T19:00:00.000Z'),
+    allDay: true,
+  }, {
+    text: 'Prepare 2021 Marketing Plan',
+    startDate: new Date('2021-04-01T18:00:00.000Z'),
+    endDate: new Date('2021-04-01T20:30:00.000Z'),
+  }, {
+    text: 'Brochure Design Review',
+    startDate: new Date('2021-04-01T21:00:00.000Z'),
+    endDate: new Date('2021-04-01T22:30:00.000Z'),
+  }, {
+    text: 'Create Icons for Website',
+    startDate: new Date('2021-04-02T17:00:00.000Z'),
+    endDate: new Date('2021-04-02T18:30:00.000Z'),
+  }, {
+    text: 'Upgrade Server Hardware',
+    startDate: new Date('2021-04-02T21:30:00.000Z'),
+    endDate: new Date('2021-04-02T23:00:00.000Z'),
+  }, {
+    text: 'Submit New Website Design',
+    startDate: new Date('2021-04-02T23:30:00.000Z'),
+    endDate: new Date('2021-04-03T01:00:00.000Z'),
+  }, {
+    text: 'Launch New Website',
+    startDate: new Date('2021-04-02T19:20:00.000Z'),
+    endDate: new Date('2021-04-02T21:00:00.000Z'),
+  },
+];
+
+const SCROLLBAR_STYLES = `
+    ::-webkit-scrollbar {
+      -webkit-appearance: none;
+      width: 7px;
+    }
+    ::-webkit-scrollbar-thumb {
+      border-radius: 4px;
+      background-color: rgba(0, 0, 0, .5);
+      -webkit-box-shadow: 0 0 1px rgba(255, 255, 255, .5);
+    }
+    .dx-scheduler-date-table-scrollable .dx-scrollable-container {
+      overflow: scroll !important;
+    }
+`;
+
+const scrollHorizontallyToEnd = ClientFunction(() => {
+  const scheduler = document.querySelector('.dx-scheduler');
+  if (!scheduler) return { success: false, message: 'Scheduler not found' };
+
+  const dateTableScrollable = scheduler.querySelector('.dx-scheduler-date-table-scrollable');
+  if (!dateTableScrollable) return { success: false, message: 'Date table scrollable not found' };
+
+  const scrollableContainer = dateTableScrollable.querySelector('.dx-scrollable-container');
+  if (!scrollableContainer) return { success: false, message: 'Scrollable container not found' };
+
+  const initialScrollLeft = scrollableContainer.scrollLeft;
+  const maxScrollLeft = scrollableContainer.scrollWidth - scrollableContainer.clientWidth;
+
+  scrollableContainer.scrollLeft = maxScrollLeft;
+
+  return {
+    success: true,
+    initialScrollLeft,
+    maxScrollLeft,
+    finalScrollLeft: scrollableContainer.scrollLeft,
+    hasHorizontalScroll: maxScrollLeft > 0,
+  };
+});
+
+test('Scheduler horizontal scroll screenshot test', async (t) => {
```

Comment:
Let's give this test more meaningful name, I think current name is very generic. Also we can mention the ticket number in the test name